oviewer/ovstyle.go

```diff
@@ -138,3 +140,18 @@ func applyStyle(style tcell.Style, s OVStyle) tcell.Style {
 
 	return style
 }
+
+// underLineStyle sets the underline style.
+// only support 0-5.
+// 0: None, 1: Single, 2: Double, 3: Curly, 4: Dotted, 5: Dashed
+func underLineStyle(ustyle string) tcell.UnderlineStyle {
+	n, err := strconv.Atoi(ustyle)
+	if err != nil {
+		return 0
+	}
+	if tcell.UnderlineStyle(n) > tcell.UnderlineStyleDashed {
```

Comment:
This looks strange

I assume you only wanto support 0-5 range, but these are iota, I assume. So comparing them is a but strange

I would use a switch with the exact value you support in one case.

Also, if you only supports the value "0", "1", maybe you could avoid the string to int conversion

Reply:
I saw you added negative checks now

But what do you think about my comment just above this one?

@nobe4

Reply:
This is how to do the behavior of an unauthorized value outside the specifications.
The operation changes with Terminal-Emulator, so it may change in the future.

Originally, it is derived from the escape sequence specification, so it doesn't matter much whether it is an iota.

noburus is not a person concerned. 😥

Reply:
Wow, a double wrong ping 🤦‍♂️